packages/core/lib/core.js

```diff
@@ -7,12 +7,13 @@ const { getConfig } = require('./config');
 const { environmentalize } = require('./env');
 
 exports.Mixin = class Mixin {
-  constructor(config) {
+  constructor(config, options) {
     this.config = config;
+    this.options = options;
   }
 };
 
-exports.bootstrap = function bootstrap(overrides, ...args) {
+exports.bootstrap = function bootstrap(overrides = {}, options = {}) {
```

Comment:
Unrelated to this PR, but the name `overrides` is unclear. How about `configOverrides`?